read-fonts/src/tables/gpos/closure.rs

```diff
@@ -0,0 +1,23 @@
+//! support closure for GPOS
+
+use super::Gpos;
+use crate::{collections::IntSet, ReadError, Tag};
+
+impl Gpos<'_> {
+    /// Return a set of all feature indices underneath the specified scripts, languages and features
+    ///
+    /// if no script is provided, all scripts will be queried
+    /// if no language is provided, all languages will be queried
+    /// if no feature is provided, all features will be queried
+    pub fn collect_features(
+        &self,
+        scripts: Option<&IntSet<Tag>>,
```

Comment:
Instead of using Option to indicate matching all you can just rely on using inverted sets. Passing IntSet::<Tag>::all() should result in the same behaviour as None.